docs/sources/k6/next/get-started/running-k6.md

```diff
@@ -33,7 +33,7 @@ To run a simple local script:
    ```
 
    ```docker
-   $ docker run --rm -i -v $PWD:/app -w /app grafana/k6 new
+   $ docker run --rm -i --user $UID -v $PWD:/app -w /app grafana/k6 new
```

Comment:
```suggestion
   $ docker run --rm -i --user $UID -v $PWD:/app -w /app grafana/k6 new
```
This is bash dependent, should we use instead `$(id -u)`? @oleiade Does it work for you on fish?

Reply:
Good point. In fish I have to quote "$ENV_VARIABLE", but `--user "$UID"` does work. Let's not bother too much about fish because it has its set of differences with standard shells like `$(id -u)` would be `(id -u)` anyway.

I do think switching to `$(id -u)` to be a good idea nonetheless anyway 👍🏻 (fish users will know what to do about it 🙇🏻